File: packages/cli/test/fixtures/ranking-corpora/java/src/main/java/com/example/ranking/handlers/UpdateHandler.java
package com.example.ranking.handlers;

import com.example.ranking.util.Logger;

/** Changes an existing user's display name. */
public class UpdateHandler {
    public void updateUser(int id, String name) {
        if (id <= 0) {
            Logger.logError("updateUser", new IllegalArgumentException("id must be positive"));
            return;
        }
        Logger.logInfo("updated user " + name);
    }
}

File: packages/cli/test/fixtures/ranking-corpora/kotlin/src/main/kotlin/com/example/ranking/handlers/UpdateHandler.kt
package com.example.ranking.handlers

import com.example.ranking.util.Logger

/** Changes an existing user's display name. */
class UpdateHandler {
    fun updateUser(id: Int, name: String) {
        if (id <= 0) {
            Logger.logError("updateUser", IllegalArgumentException("id must be positive"))
            return
        }
        Logger.logInfo("updated user $name")
    }
}
